profiles/coreos/arm64/package.accept_keywords

```diff
@@ -13,16 +13,13 @@
 
 =dev-lang/yasm-1.3.0-r1 ~arm64
 =dev-libs/libassuan-2.5.1 ~arm64
-=dev-libs/libevent-2.1.8 ~arm64
 =dev-libs/libusb-1.0.21 ~arm64
 
-=dev-perl/Text-Unidecode-1.270.0 ~arm64
-
 # needed to force enable bpftool for arm64
 =dev-util/bpftool-5.15.8 **
 
 # needed by arm64-native SDK
-dev-util/checkbashisms *
+=dev-util/checkbashisms-2.21.4 ~arm64
```

Comment:
mental note: we should open a stabilization request for ARM64 after the next alpha release.

Reply:
Yeah, there will be a bunch of packages to stabilize. Some script that detects such packages would be useful, but one thing at a time.

Reply:
yeah, I currently note them into my TODO list and open a stabilization request from time to time.